app/app.py

```diff
@@ -0,0 +1,76 @@
+"""
+Main training task.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+
+import os, json, random
+
+from flask import Flask, render_template, jsonify, request, send_from_directory
+from annoy import AnnoyIndex
+
+
+### Util Functions
+def get_json(filename):
```

Comment:
It's pretty idiomatic to inline this since it's only 2 lines and self-explanatory.